package.json

```diff
@@ -42,6 +42,7 @@
     "gatsby-source-contentful": "^2.0.3",
     "gatsby-source-filesystem": "^2.0.3",
     "gatsby-transformer-remark": "^2.1.7",
+    "gsap": "^2.0.2",
```

Comment:
Don't think you ended up using this?

Also, it looks like the `yarn.lock` got trashed and `package-lock.json` got added. Need to revert that and only use yarn.

Reply:
I did not use the gsap package. I was going to try adding animation to the loading of the FAQs but didn't get around to it. 

As far as the yarn vs. npm deal, not sure exactly what needs to be done so may need some guidance there. Same with the `yarn.lock` vs.` package-lock.json` issue.